src/neuroglancer/neuroglancer

```diff
@@ -0,0 +1 @@
+/Users/roseva1/Desktop/Dashbaord/BossDB/Neuvue/neuroglancer/src/neuroglancer
```

Comment:
can we remove this for the PR

Reply:
@Raphtor can we just delete this and the rest of the neuroglancer build will be fine?